src/org/usfirst/frc/team2399/robot/commands/JoyDrive.java

```diff
@@ -11,21 +11,22 @@
  */
 public class JoyDrive extends Command {
 
-	private OI oi = Robot.oi;
 	private DriveTrain driveTrain = Robot.driveTrain;
 
 	public JoyDrive() {
 		requires(driveTrain);
+		setInterruptible(true);
 	}
-
+	
```

Comment:
As usual, minor thing, but everyone's Eclipse should be set to automatically remove trailing whitespace on save. `git blame` tells me this one was Andrew